examples/Encrypted Inference-LinearRegression.ipynb

```diff
@@ -0,0 +1,1166 @@
+{
```

Comment:
<p>MAYBE : assign int(len(features))*train_test_split to a variable, for code clarity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/SyMPC/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>